client/navigation/connected-left-nav.jsx

```diff
@@ -34,7 +34,7 @@ function stateToProps(state) {
   return {
     activeGame: state.activeGame,
     auth: state.auth,
-    chatChannels: state.chat.channels.map(c => ({
+    chatChannels: Array.from(state.chat.channels.values()).map(c => ({
```

Comment:
this is already an array, no? You can just call `map` directly on it. (If it's *not* and you still need to use `Array.from`, you can remove the call to `map` and just use the second parameter of `Array.from` to do the mapping)

Reply:
No, it's a `Set`; see the comment above.